### ISSUE TITLE
per-directory-history-toggle-history 

### ISSUE THREAD

**Issue by theova** [jimhester/per-directory-history#29]
Hi,

**Describe the bug**
when running `$ per-directory-history-toggle-history` I get the following error:
```bash 

using global historyper-directory-history-toggle-history:zle:8: widgets can only be called when ZLE is active
per-directory-history-toggle-history:zle:9: widgets can only be called when ZLE is active
```
Furthermore, I am not able to toggle the history.

**To Reproduce**
1. Enable plugin 'per-directory-history'
2. Run command `per-directory-history-toggle-history`
3. See error

**Expected behavior**
The functions works as expected and no error is thrown.

**System:**
 - OS: Parabola GNU/Linux-libre
 - Zsh version:. 5.7.1
 - Terminal emulator: Termite

Thank you very much for this amazing plugin :-)

**Comment by jimhester** [comment]
The function is meant to work as a [zle widget](http://zsh.sourceforge.net/Doc/Release/Zsh-Line-Editor.html#Zle-Widgets), not as a normal function. Generally you use it by binding it to a key rather than calling it directly.

Possibly the plugin could be changed to accommodate both interfaces, but this is not currently how it is designed.

**Comment by theova** [comment]
Thank you for your answer :-)

**Comment by CharlesARoy** [comment]
Could either of you elaborate on how to "set the PER_DIRECTORY_HISTORY_TOGGLE environment variable" so that I can use something other than ^G to toggle per-directory-history?

I tried setting:
`export PER_DIRECTORY_HISTORY_TOGGLE='^H'`
and later
`export PER_DIRECTORY_HISTORY_TOGGLE=^H`
for good measure, but neither worked.

Originally I tried creating an alias like:
`alias pd='per-directory-history-toggle-history'`

but that resulted in the same error messages that started this issue which is what brought me here.
